server/fetcher.go

```diff
@@ -94,6 +104,104 @@ func (p *Plugin) fetchCommitsFromRepo(org, repo string, since, until time.Time)
 	return result, nil
 }
 
+func (p *Plugin) fetchContributorsFromOrg(org string) ([]*repoContributors, error) {
```

Comment:
```suggestion
func (p *Plugin) fetchContributors(org string) ([]*repoContributors, error) {
```
I think the shorter name is fine, as you are passing in org as a variable input.